CMakeLists.txt

```diff
@@ -178,3 +178,10 @@ add_test(
         ${CMAKE_BINARY_DIR}/bin/divecli extract -o .
         ${CMAKE_SOURCE_DIR}/tests/traces/bloom-frame-0080-compressed.rd
 )
+
+add_test(
+    NAME Regression_StackOverflow_MacOS
```

Comment:
nit: sorry, can you add some comments on this test to explain what it does and what bug it is trying to detect (maybe with b/number) just to be more descriptive so that in the future we know what is this test for. :)

Reply:
done.